### DISCUSSION TITLE
Criterion does not output anything when run on Windows 10.

### DISCUSSION THREAD

**Discussion by serdonke** [Snaipe/Criterion#466] [Q&A] [answered]
I followed the steps for setup on [readtheocs](https://criterion.readthedocs.io/en/master/setup.html#building-from-source).
Meson setup succeeded 
![image](https://user-images.githubusercontent.com/51852446/207467681-a3e5b52e-4e0d-4ca0-a9dd-8b7fa27780aa.png)
Ninja build succeeded
![image](https://user-images.githubusercontent.com/51852446/207467848-615c97f7-079a-43b5-80e3-2c9a0bc02534.png)

Then I copy pasted the example from [Getting Started](https://criterion.readthedocs.io/en/master/starter.html) section 
```c
#include <string.h>
#include <criterion/criterion.h>

Test(sample, test)
{
    cr_expect(strlen("Test") == 4, "Expected \"Test\" to have a length of 4.");
    cr_expect(strlen("Hello") == 4, "This will always fail, why did I add this?");
    cr_assert(strlen("") == 0);
}
```

and compiled it using 

```sh
gcc test.c -I .\Criterion\include -L .\Criterion\build\src -lcriterion
```

The resulting executable did not output anything.

![image](https://user-images.githubusercontent.com/51852446/207469324-535ce60c-62e4-4587-a718-2b7f969b331a.png)

Is there a step or flag I am missing, or am I doing everything wrong?

Here's the output of `meson build` in case its relevant.
![image](https://user-images.githubusercontent.com/51852446/207471582-e3b0279f-0516-4fcc-8f5f-e2e3ad758cac.png)

**Comment by MrAnno** [comment]
Do you have the directory containing `criterion.dll` in your `PATH`?

**Reply by serdonke** [reply]
No but I pass the path through compiler flags.

**Reply by MrAnno** [reply, marked as answer]
`RPATH` does not really exist on Windows, so you either need to set a PATH too as a runtime path or copy `criterion.dll` next to the test binaries.
